course_inventory/inventory.py

```diff
@@ -246,7 +246,7 @@ def run_course_inventory() -> Sequence[Dict[str, Union[ValidDataSourceName, pd.T
 
     # Record data source info for UDW
     udw_meta_df = pd.read_sql('SELECT * FROM unizin_metadata;', udw_conn)
```

Comment:
we may just retrieve the canvasdatadate row only:

select * from unizin_metadata where key='canvasdatadate'

Reply:
Done!